### PR TITLE
Related to item #79

### PR DESCRIPTION
**Pull request by laurie-tyz**
Some substitution of patch or fix for remediation or remediated.

one deletion in 047: deleted "applier".

### CODE REVIEW

**Review by j--- — CHANGES_REQUESTED** [review]
I think we should try to get all the relevant instances of "patch" with this issue if it's part of 79. So in 040_treesForVulMgmt.md, the headings "supplying patches" etc. and the language in those sections needs to be updated to reflect we are talking about mitigations in general with SSVC. Currently those sections imply SSVC is only about patches.

**Comment by laurie-tyz** [comment]
I don't believe these changes will fully close out the "patch" or "fix" issue.  I wanted to start slowly with *some* changes that appear obvious.  So a partial solution.  The rest of the plan was to start from the beginning (again) and identify those patch/fix instances that should really be mitigation/remediation.

Your call on how you want it done.

**Review by j--- — APPROVED** [review]
OK, if that's the plan then this looks good to me.

File: doc/version_1/047_treesForVulMgmt_4.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Once the decision points are selected and the prioritization labels agreed upon,
 Making the decision process accessible has a lot of benefits.
 Unfortunately, it also makes it a bit too easy to overcomplicate the decision.
 
-The SSVC version 1 ~applier~ deployer tree had 225 rows when we wrote it out in long text form.
+The SSVC version 1 deployer tree had 225 rows when we wrote it out in long text form.
```
Comment (j---) [inline doc/version_1/047_treesForVulMgmt_4.md:74]: I had this in strikeout (~tilde~) on purpose to make it clear that it was called something else in v1, so if a reader goes to look at the v1 doc they know it was called applier then, though we have changed it to deployer now. So I think this should stay. Sorry, I didn't catch this earlier.

**Pull request merged**